firmware/Makefile

```diff
@@ -1,4 +1,5 @@
 PROJECT_NAME := firmware
 
+IDF_PATH ?= ../esp-idf
```

Comment:
Small comment here - `setenv.sh` also adds other environment variables, but `make` finishes without errors for me with just this (and the correct `PATH` with the espressif stuff), so I don't know if we need the rest.

Reply:
Love it, you can probably get rid of the set_env.sh entirely indeed.

Reply:
There you go ;)